pydocx/openxml/markup_compatibility/fallback.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+
+
+class Fallback(XmlModel):
+    XML_TAG = 'Fallback'
+    children = XmlCollection('wordprocessing.Picture')
```

Comment:
I will be dealing with that in #204 as I mentioned during stand up :)

I don't plan to push a new release until both this and 203 are done.